wolvic/browser/vr/wvr_manager.cc

```diff
@@ -477,6 +528,10 @@ WvrManager::GetInputSourceState() {
 
     input_sources.push_back(std::move(input_source));
   }
+
+  memcpy(controller_state_, wvr_api_->get_system_state().controllerState,
+         sizeof(controller_state_));
```

Comment:
The types are the same.

WRT to the assign copy operator, yeah not a bad idea, but we're talking about some structures defined in [moz_external_vr](https://github.com/Igalia/wolvic/blob/main/app/src/main/cpp/moz_external_vr.h) which is a exact copy of the file that is distributed by Gecko. There must be a very good reason to change the structures defined there because that would create a delta with upstream.